.github/CONTRIBUTING.md

```diff
@@ -1,61 +1,74 @@
 # Contributing
-When contributing to this repository, please first discuss the change you wish to make by creating
-an issue. Please note we have a code of conduct. Please follow it in all your interactions
-with the project.
+
+When contributing to this repository, please first discuss the change you wish
+to make by creating an issue. Please note we have a code of conduct. Please
+follow it in all your interactions with the project.
 
 ## Pull Request Process
-* Create a fork of this repository, then clone the fork to your local development environment: `git clone https://github.com/YOUR_USERNAME/styled-components-modifiers.git`
-* Create a branch with a meaningful name reflecting the bug fix or new feature: `git checkout -b my-new-feature`
-* Make your changes (including updates/additions to tests) and commit: `git add` and `git commit`
-* Make sure that the tests still pass: `npm run review`
-* Push your branch: `git push -u origin my-new-feature`
-* [Submit a pull request](https://github.com/Decisiv/styled-components-modifiers/compare) to the upstream `styled-components-modifiers` repository.
-* Enter a descriptive title for the pull request and fill in the PR template, describing the proposed changes.
-* Wait for a maintainer to review your PR.  Follow up on any comments from the reviewer, and wait for a maintainer to merge the PR.
+
+- Create a fork of this repository, then clone the fork to your local
+  development environment:
+  `git clone https://github.com/YOUR_USERNAME/styled-components-modifiers.git`
+- Create a branch with a meaningful name reflecting the bug fix or new feature:
+  `git checkout -b my-new-feature`
+- Make your changes (including updates/additions to tests) and commit: `git add`
+  and `git commit`
+- Make sure that the tests still pass: `npm run review`
```

Comment:
This document specifies the use of `npm` but the project has a `yarn.lock` and generally we use `yarn`. The docs should probably be updated to `yarn` instead of `npm`?